modules/library/metadata/src/main/java/org/geotools/util/SoftValueHashMap.java

```diff
@@ -44,11 +45,11 @@
  * collection. The amount of entries to retain by hard reference is specified at {@linkplain
  * #SoftValueHashMap(int) construction time}.
  * <p>
- * This map is thread-safe. It accepts the null key, but doesn't accepts null values. Usage
- * of {@linkplain #values value}, {@linkplain #keySet key} or {@linkplain #entrySet entry}
- * collections are supported except for direct usage of their iterators, which may throw
- * {@link java.util.ConcurrentModificationException} randomly depending on the garbage collector
- * activity.
+ * This map is thread-safe. Since it uses the concurrent structures {@linkplain ConcurrentHashMap}
+ * and {@linkplain ConcurrentLinkedQueue}, such structures do not accept either the null key and null values, and 
```

Comment:
The usage of the placeholder should be a internal detail, no need to tell clients about it. Or do they need to know for any reason?